docs/nb/bgs_main_map.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    orig_density_per_deg = 2500
</pre><p>Let's put this as rand_density_per_deg2 in e.g. geometry.py and import</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahbigwood/S4Mock/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>